libs/components/phone-field/testing/src/phonefield/phone-field-fixture.spec.ts

```diff
@@ -198,29 +212,34 @@ describe('PhoneField fixture', () => {
       'selectedCountryChange'
     );
 
-    // search for a country by name
-    const results = await phonefieldFixture.searchCountry(COUNTRY_AU.name);
+    if (COUNTRY_AU.name) {
```

Comment:
is this correct? i'm still confused as to why this is question if it exists, since it's a const that is being declared above. i know it's typed as optional, but i figured the const declaration would take precedence.